include/openenclave/corelibc/sys/bits/utsname.h

```diff
@@ -10,9 +10,5 @@ struct __OE_UTSNAME
     char release[__OE_UTSNAME_FIELD_SIZE];
     char version[__OE_UTSNAME_FIELD_SIZE];
     char machine[__OE_UTSNAME_FIELD_SIZE];
-#ifdef _GNU_SOURCE
```

Comment:
This is technically fine, but I'll channel my inner @anakrish here and point out it's not actually part of the POSIX standard, for a set of features going by the "POSIX" name.

Recognizing the distinction is useful particularly as it comes to documentation; it's a much clearer statement to say our libraries only depend on POSIX, but any additional GNU extensions that end up becoming a dependency require extra documentation and expands the surface Windows needs to implement for parity.

Reply:
I assume you are referring to the *domainname** field, not **uname()**, which is in the POSIX standard. The **domainname** field is required to make the structure align with the MUSL structure. We could introduce a separate OCALL for gethostname. That might be cleaner.